### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Harshtroo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+*.py[co]
+*.sw[po]
+*.egg*
+*.rdb
+.idea/
+.coverage
+pip-log.txt
+docs/_gh-pages
+build/
```
Comment (yatishTT) [inline .gitignore]: why unwanted expressions are there

File: requirements/base.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Django 
+Django>=4.1,<4.3
+django-kn-defaults==0.3.2
+djvue==0.2.1
+sentry-sdk==1.11.1
+elastic-apm==6.13.2
```
Comment (yatishTT) [inline requirements/base.txt]: un wanted requiremnts

File: tms/tms/settings/production.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import logging
+
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
```
Comment (yatishTT) [inline tms/tms/settings/production.py]: where you are using sentry ?

File: tms/tms/settings/production.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+from .base import *  # noqa
+from .base import env
+
+# SECURITY
+# ------------------------------------------------------------------------------
+# https://docs.djangoproject.com/en/dev/ref/settings/#secure-proxy-ssl-header
+SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+# https://docs.djangoproject.com/en/dev/ref/settings/#secure-ssl-redirect
+SECURE_SSL_REDIRECT = env.bool("DJANGO_SECURE_SSL_REDIRECT", default=True)
+# https://docs.djangoproject.com/en/dev/ref/settings/#session-cookie-secure
```
Comment (yatishTT) [inline tms/tms/settings/production.py]: unnecessary settings

**Review by yatishTT — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+.vscode/*
+
+
+tms/utils/
+tms/user/
+tms/conftest.py
+tms/docker-compose.yml
+tms/Dockerfile
+tms/tms.db
+tms/utils
```
Comment (yatishTrootech) [inline .gitignore]: why utils ?

**Review by yatishTrootech — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 5 above)
Comment (Harshtroo) [reply .gitignore]: because right now utils not use so

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+*.sw[po]
+*.egg*
+*.rdb
+.idea/
+pip-log.txt
+tmp/*
+*.log.*
+*.log
+*.db
+db/
```
Comment (yatishTT) [inline .gitignore]: only expression related to your project should be in this file

**Review by yatishTT — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,19 @@
+*.py[cod]
+__pycache__/
+.idea/
```
Comment (yatishTrootech) [inline .gitignore:4]: *.py[cod]
__pycache__/
.idea/
tmp/*
*.log.*
*.log
tms/media/*
.idea/*
tms/conftest.py
tms/tms.db

**Review by yatishTrootech — CHANGES_REQUESTED** [review]

**Pull request merged**